requirements.txt

```diff
@@ -0,0 +1,5 @@
+django==3.0.2
+django-mongodb-engine==0.6.0
+djangotoolbox==1.8.0
+pylint==2.4.4
+pymongo==3.10.1
```

Comment:
This should sit within `django-backend` file since this is only related to Python